.idea/libraries/Maven__com_beust_jcommander_1_72.xml

```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```

Comment:
`.idea` directory normally is not commited

Reply:
removed